test/fixtures/shared/variables.tf

```diff
@@ -14,15 +14,24 @@
  * limitations under the License.
  */
 
-variable "project_id" {}
+variable "project_id" {
+  description = "The GCP project to use for integration tests"
+}
 
-variable "credentials_path_relative" {}
+variable "credentials_path_relative" {
+  description = ...
```

Comment:
This is not a proper description.

Reply:
Yep, sorry -- accidentally pushed an interstitial state. Fixed.